test/src/test/java/org/apache/accumulo/harness/SharedMiniClusterIT.java

```diff
@@ -48,6 +50,7 @@
  * a method annotated with the {@link org.junit.BeforeClass} JUnit annotation and {@link #stopMiniCluster()} in a method annotated with the
  * {@link org.junit.AfterClass} JUnit annotation.
  */
+@Category(MiniClusterOnlyTest.class)
```

Comment:
Ah, neat. I didn't realize java had inheritance of annotations at all. I guess it's just for interfaces.